nixos/modules/services/misc/sourcehut/builds.nix

```diff
@@ -161,6 +160,14 @@ in
           partOf = [ "buildsrht.service" ];
           description = "builds.sr.ht worker service";
           path = [ pkgs.openssh pkgs.docker ];
+          preStart = ''
+            # create and load qemu:latest image for docker
```

Comment:
This seems rather expensive. Loading the entire image on every startup? Also are we sure the images are compatible? I am sure the original image provides some fancy entry point script that we don't

Reply:
> This seems rather expensive. Loading the entire image on every startup?

I can add version check.
https://github.com/NixOS/nixpkgs/blob/5d6b601e171cddda680314dc8bad5b8f41068338/nixos/modules/services/misc/sourcehut/service.nix#L54
Is this approach would be acceptable? 

> I am sure the original image provides some fancy entry point script that we don't

It doesn't have any entry point scripts: https://git.sr.ht/~sircmpwn/builds.sr.ht/tree/master/item/images/qemu/Dockerfile